src/VCR/Configuration.php

```diff
@@ -121,16 +121,26 @@ class Configuration
     private $blackList = array('src/VCR/LibraryHooks/', 'src/VCR/Util/SoapClient', 'tests/VCR/Filter');
 
     /**
-     * The mode which determines how requests are handled
-     *
-     * Currently supported modes:
-     *      - new_episodes (Always allows new HTTP requests)
-     *      - once (Will allow new HTTP requests the first time the cassette is created then throw an exception after that)
-     *      - none (Will never allow new HTTP requests)
+     * Always allow to do HTTP requests and add to the cassette. Default mode.
+     */
+    const MODE_NEW_EPISODES = 'new_episodes';
```

Comment:
constants is not strictly related to the fix, nor needed by it. but i think it would be better to use constants for this instead of handling around strings that can be mistyped.